### ISSUE TITLE
Ability to clear performance chart

### ISSUE THREAD

**Issue by brettacook** [GoldenCheetah/GoldenCheetah#1818]
Before starting a subsequent workout, it would be nice to be able to select an item from the 'view' menu to clear the performance chart data (the one with HR, power, cadence, etc), as well as any other charts prior to starting the next workout.  Otherwise, the display starts to look a little cluttered.

**Comment by liversedge** [comment]
hit stop.

**Comment by brettacook** [comment]
Ummm, I hit stop, which clears all but the blue line (the blue line stays
on the graph all the time).

**Comment by liversedge** [comment]
The blue line is the workout selected in the sidebar.

**Comment by brettacook** [comment]
Hmmm - I don't have a sidebar (I close it) and I don't remember selecting a
"workout", but I'll take a look tonight.

**Comment by brettacook** [comment]
So I took a look today and still can't find what you're referring to.
There are now 4-5 different blue lines on the screen, and they do indeed
show old workouts.  However, there is not list of workouts in the sidebar
that corresponds:  I only have two items in the "Workouts" section, both
labeled "Manual Erg Mode".  Neither is selected and I can't find a way to
delete these.
If you could clarify what you mean by "sidebar" or where I should be able
to see these "workouts", I'd really appreciate it.

**Comment by liversedge** [comment]
Can you post a screenshot.

**Comment by brettacook** [comment]
LOL - was just getting to that when you responded.  Sorry that I didn't think of it sooner :-)
![goldencheetahblueline](https://cloud.githubusercontent.com/assets/16944189/13549850/43335246-e2cc-11e5-9a0b-a472a6436a49.jpg)

**Comment by liversedge** [comment]
ewww.

**Comment by brettacook** [comment]
Yeah, that's what I said.  Do you see something that I'm doing wrong, or something obvious that I'm missing?  As you said, it's really not pretty to look at.

**Comment by brettacook** [comment]
Question - since it sounds like this is something that hasn't been identified before, should I re-post the question/bug (so that it's open)?

**Comment by brettacook** [comment]
I could not figure out how to reopen this issue (or re-flag it as a bug), so I opened a new issue.  Thanks for taking a look.

**Comment by liversedge** [comment]
The issue will be resolved since that chart is being deprecated for the workout editor anyway.

**Comment by brettacook** [comment]
Do you know when that change is going to occur?
Thanks again for taking a look at it.

**Comment by liversedge** [comment]
its already in the dev builds.
